schema/proto3/vpc.proto

```diff
@@ -31,10 +31,6 @@ message VpcConfiguration {
     string name = 4;
     string cidr = 5;
     int64 tunnel_id = 6;
-    // FIXME: MTU info would be part VPC right?
```

Comment:
That is right. MTU will be on vpc level. 

Does this mean that they will use the same MTU on a region basis and ignore MTU value from Neutron Server.

Reply:
The MTU setting depends on the physical NIC MTU setting and all the networking hardware needs to support it (e.g. jumbo frame MTU=9000). Therefore, we cannot simply allow a VPC to set certain MTU. More detail here: https://github.com/openstack/neutron/blob/cc69828ff09f5dbc37488f3f56bb045772e35634/doc/source/admin/config-mtu.rst#L7

Reply:
Thanks for the clarification. The document makes sense but somehow contradicts with the other document. OpenStack Neutron v2.0 API allows to puts a MTU field when creating a network.

https://docs.openstack.org/api-ref/network/v2/?expanded=create-network-detail#create-network

{
    "network": {
        "name": "sample_network",
        "admin_state_up": true,
        "dns_domain": "my-domain.org.",
        "qos_policy_id": "6a8454ade84346f59e8d40665f878b2e",
        "mtu": 1400
    }
}

Reply:
Looking deeper into MTU in neutron code, while it is not being used in ovs neutron agent where it is configuring ports created by nova agent, MTU setting are being used in dvr edge router (for source NAT), dvr floating IP management and neutron L3 agent (I am guessing it is because these component will create (e.g. gateway) ports). We will need to look into MTU topic more when we design our router.

Reply:
I see. Let us keep this field for now which allows us to revisit for L3 networking staff shortly.